creator-node/src/monitors/MonitoringQueue.js

```diff
@@ -63,6 +65,16 @@ class MonitoringQueue {
     )
   }
 
+  /**
```

Comment:
This makes sense to me, but maybe we should make this blocking. Either way though, we shouldn't be re-initting this much to cause problems

Reply:
i want to remove the dependency completely from monitors and have a way for this middleware to directly query this value. monitors is intended to be informational and we shouldn't have coupled these but for now i'm noticing cnodes taking minutes to the storage space values on first init so i'm trying this as a stopgap

Reply:
Sounds good to me